packages/keyring-snap-bridge/src/SnapKeyring.test.ts

```diff
@@ -656,7 +656,7 @@ describe('SnapKeyring', () => {
           `Method '${EthMethod.PersonalSign}' not supported for account ${ethEoaAccount1.address}`,
         );
         // Restore `EthMethod.PersonalSign` and remove `EthMethod.SignTransaction`
-        updatedMethods = Object.values(EthMethod).filter(
```

Comment:
`EthMethod` was not truly correct here, since we use an EOA account here, which should use `ETH_EOA_METHODS` (since `EthMethod` also contains method for ERC4337 accounts)